### PR TITLE
Fix #1540

### LINKED ISSUE #1540: [Migration] Transform attributes in calculation views
### Target
1. Transform the `associatedObjectUri` to point to a synonym. See example how it looked before [here](https://github.com/SAP-samples/hana-structurer-one/blob/main/Models/SAP_CA_TWEETB.calculationview#L76). This would mean from `/Summit15.Models/calculationviews/SAP_DIM_TWEETINFOB` to `Summit15.Models:: SAP_DIM_TWEETINFOB`
2. Transform the `measureMapping` to omit the schema name.  See example [here](https://github.com/SAP-samples/hana-structurer-one/blob/main/Models/SAP_AN_TWEET.analyticview#L43) even though that's analytics view. The `schemaName` must be removed

### PR DESCRIPTION
**Pull request by dimitarquanterall**
Signed-off-by: dimitarquanterall <d.aleksandrov@sap.com>

Fix #1540 - transforms associatedObjectUri and measureMapping

### CODE REVIEW

File: modules/engines/engine-core/src/main/java/com/sap/xsk/modificators/CalculationViewDataSourceTransformation.java [hunk 1]
```diff
@@ -30,6 +30,14 @@ public class CalculationViewDataSourceTransformation {
...
       + "    </xsl:template>\n"
+      + "    <xsl:template match=\"logicalJoin/@associatedObjectUri\">\n"
+      + "         <xsl:attribute name=\"associatedObjectUri\">\n"
+      + "           <xsl:value-of select=\"concat(substring-before(substring-after(., '/'), '/'), '::', substring-after(substring-after(substring-after(., '/'), '/'), '/'))\" disable-output-escaping=\"yes\"  />\n"
+      + "         </xsl:attribute>\n"
+      + "     </xsl:template>\n"
+      + "     <xsl:template match=\"measureMapping/@schemaName\">\n"
+      + "         <xsl:copy-of select=\"/@*[name(.)!='schemaName']|node()\" />\n"
+      + "     </xsl:template>\n"
       + "</xsl:stylesheet>";
```
Comment (ivanvolkoff) [inline modules/engines/engine-core/src/main/java/com/sap/xsk/modificators/CalculationViewDataSourceTransformation.java:42]: i think would be better to rename CalculationViewDataSourceTransformation -> CalculationViewTransformation as from now we would make changes not only on data source artefact.

**Review by ivanvolkoff — CHANGES_REQUESTED** [review]
small changes required

File: modules/engines/engine-core/src/main/java/com/sap/xsk/modificators/CalculationViewDataSourceTransformation.java (hunk 1 above)
Comment (dimitarquanterall) [reply modules/engines/engine-core/src/main/java/com/sap/xsk/modificators/CalculationViewDataSourceTransformation.java:42]: Ok. Committed.

**Review by ivanvolkoff — DISMISSED** [review]

**Comment by dpanayotov** [comment]
depends on #1628 to resolve issue with tests

**Review by vmutafov — CHANGES_REQUESTED** [review]
Please, add JUnit tests

**Review by vmutafov — APPROVED** [review]

**Review by ivanvolkoff — APPROVED** [review]

**Pull request merged**